src/components/ProfilePage.tsx

```diff
@@ -0,0 +1,221 @@
+                        <button onClick={()=>setEditing(true)}>Edit Public Profile</button> 
+                    </>
+                    :
+                    <>
+                        <button onClick={()=>updateUserProfile()}>Save Changes</button>
+                        <button onClick={onCancel}>Cancel</button>
+                    </> 
+                    }
+                </div>
+                <Row>
+                    <Col sm={3} >
+                        {validProfilePic}   
+                    </Col>
+                    <Col sm={9}>
+                        <div className='grid-container'>
+                            <div className='containers'>
+                                {!!!editing ? 
+                                    <Typography variant='headline3'>{`${userProfile?.firstName} ${userProfile?.lastName}`}</Typography>
+                                : 
+                                <FormGroup style={{display:'inline-block'}}>
+                                    {EditField('First name', firstName, setFirstName)}
+                                    {EditField('Last name', lastName, setLastName)}
+                                </FormGroup>
+                                }
+                            </div>
+                            <div className='containers'>
+                                {!!!editing ? 
+                                <div>
+                                    {userProfile?.position} <br/>
+                                    {userProfile?.company} <br/>
+                                    {userProfile?.location}
+                                </div> : 
+                                <FormGroup>
+                                    {EditField('Position', position, setPosition)}
+                                    {EditField('Company', company, setCompany)}
+                                    {EditField('Location', location, setLocation)}
+                                </FormGroup>   
+                            }
+                            </div>
+                            <div className='containers'>
+                                {!!!editing ? userProfile?.summary : 
+                                <FormGroup>
+                                <FormLabel>Summary</FormLabel>
+                                <FormControl
+                                    onChange={e=>setSummary(e.target.value)}
+                                    value={summary}
+                                    placeholder='Enter summary'
+                                    as='textarea'
+                                    rows={3}
+                                /> 
+                                </FormGroup>
+                                }
+                            </div>
+                            <div className='containers'>
+                                <a href={"mailto:" + userProfile?.userName}><img className='contact-icon' src={MailIcon}/>{userProfile?.userName}</a>
```

Comment:
I think this mailto link should be `mailto:${userProfile?.userName}@synapse.org`